README.md

```diff
@@ -10,29 +10,29 @@ It makes use of docker, and docker-compose.
 
 ## Installing
 
-[Install docker and docker-compose](https://docs.docker.com/compose/install/) (make sure you install the correct versions)
+Pre-requisites: install [Virtualbox](https://www.virtualbox.org/wiki/Downloads) first, then [Vagrant](https://www.vagrantup.com/downloads.html). Even if you have a linux box, this is *very* recommended.
+If you decide to not do it, sigh... but don't come to cry to us. [Install docker and docker-compose](https://docs.docker.com/compose/install/) (make sure you install the correct versions)
 
 Install the web application:
 ```
 git clone --recursive https://github.com/AEGEE/oms-docker.git
 cd oms-docker
-./oms.sh up 
+./start.sh 
 ```
 
-*This is assuming you are running a Linux installation, for other operating systems virtualization is recommended.*
-
-If you have a non-Linux installation, then the commands are as following (assuming you have Virtualbox and Vagrant installed as per the link below the commands)
+If you have a non-Linux installation, then the commands are as following (assuming you have Virtualbox and Vagrant installed, as per above)
 
 ```
 git clone --recursive https://github.com/AEGEE/oms-docker.git
 cd oms-docker
 vagrant up 
 ```
+**It will take a while**, so grab a cup of some hot beverage.
 
 [**More information on the installation**](https://oms-project.atlassian.net/wiki/spaces/GENERAL/pages/17235970/Installation)
 
 ## Usage
-After running the system, you can navigate to it in your web browser. The URLs differ based on how you run it; however no matter how you decide to run it, _it is suggested_ to edit the `/etc/hosts` file to add the entry: `192.168.168.168 appserver traefik.appserver portainer.appserver`, to be able to use advanced features.
+After running the system, you can navigate to it in your web browser. The URLs differ based on how you run it; however no matter how you decide to run it, _it is suggested_ to edit the `/etc/hosts` file to add the entry: `192.168.168.168 my.appserver traefik.appserver portainer.appserver`, to be able to use advanced features.
```

Comment:
1) I suggest putting other addresses as well (like kibana and my) to hosts
2) reflect here that on Windows it's `C:/Windows/system32/drivers/etc/hosts`

Reply:
1. My is already there, but kibana not. I don't think others (e.g. status, and future ones) will be relevant
2. Yesterday I wanted to get it done for the simple case, though today I see that powershell has some more advanced features! I was thinking that cmd was shit (and it is), but ps is not as bad. I will add a .ps1 file that does the same

Reply:
1. sorry, didn't notice. then it's all good

Reply:
maybe just put 'to edit the `/etc/hosts` (on Windows, it's `C:/Windows/system32/drivers/etc/hosts`)' instead to Readme and let people figure that out by themselves?